src/inspector_agent.cc

```diff
@@ -470,21 +462,19 @@ void AgentImpl::OnRemoteDataIO(uv_stream_t* stream,
     }
     DisconnectAndDisposeIO(socket);
   }
-  uv_cond_broadcast(&agent->pause_cond_);
+  agent->pause_cond_.Broadcast(scoped_lock);
```

Comment:
@pavelfeldman Can you comment on whether the changes to this function look correct to you?  ISTM it should be holding the lock when signalling the condition variable.